src/js/apps/globals/nav_app.js

```diff
@@ -412,7 +413,7 @@ export default RouterApp.extend({
           .then(() => {
             patientModal.destroy();
 
-            if (patientFormId) {
+            if (patientFormId && isForNewPatient) {
```

Comment:
I wanted to use `&& patient.isNew()` here. But that always returns `false` in this context.

Reply:
Good call, updated it to check `patientClone.isNew()`.